plugins/modules/system/xfconf.py

```diff
@@ -277,8 +277,7 @@ def state_present(self):
 
 
 def main():
```

Comment:
How about
```suggestion
main = XFConfProperty.execute
```

Reply:
That works, of course, but it does not help the readability of this code.

Reply:
Fair enough. I'd prefer calling it in the "if main" block. But it looks like it'd require disabling one linting rule (which would be fine if somebody bothered to add a new rule to the linter).